modules/@apostrophecms/ui/ui/apos/mixins/AposAdvisoryLockMixin.js

```diff
@@ -93,21 +93,14 @@ export default {
     // detect lock errors with `isLockedError` and call this method. The rest of the time,
     // it is called for you.
     async showLockedError(e) {
-      if (e.body.data.me) {
-        // We use an alert because it is a clear interruption of their
-        // work, and because a notification would appear in both windows
-        // if control was taken by the same user in another window,
-        // which would be confusing.
-        await apos.alert({
-          heading: 'You Took Control in Another Window',
-          description: 'You took control of this document in another tab or window.'
-        });
-      } else {
-        await apos.alert({
-          heading: 'Another User Took Control',
-          description: 'Another user took control of the document.'
-        });
-      }
+      // We use an alert because it is a clear interruption of their
+      // work, and because a notification would appear in both windows
+      // if control was taken by the same user in another window,
+      // which would be confusing.
+      await apos.alert({
+        heading: 'Multiple Editors',
+        description: `${e.body.data.me ? 'You' : e.body.data.title} took control of this document in another tab or window. A document can only be edited in one place at a time.`
```

Comment:
What if the person who took control is not "You", this feels very confusing to me. This could lead someone to think they'r doing something wrong. I don't think pointing out that a specific **other** user is doing it will create a 'slap fight', its just transparency. Are we able to identify the user who took control?

Reply:
This notification now reads:

"Bob Smith took control of this document in another tab or window. A document can only be edited in one place at a time."

This seems like what you're asking for. I think you might be responding to the word "title," but that's the `title` property of the user who took control (their name).